### PR TITLE
Code Review changes

### PR DESCRIPTION
**Pull request by ATMueller**
All my code review changes.

Analysis: The program looks as if it is off to a very good start. Some design philosophy flaws in terms of too much being on the screen at once, but I believe they have time to change that eventually.

The program was available as soon as the assignment came out.

The program was well commented and easy for me to understand.

The program does compile.

Concepts I learned:
1. Bootstrap can be used to make good looking sites with ease.
image

2. Using Regions in code
image

### CODE REVIEW

File: src/main/java/com/shippingcostestimator/enterprise/dto/Destination.java [hunk 1]
```diff
@@ -12,6 +12,6 @@
  */
 public @Data
 class Destination {
-    String startAddress;
-    String endAddress;
+    private String startAddress;
+    private String endAddress;
```
Comment (ATMueller) [inline src/main/java/com/shippingcostestimator/enterprise/dto/Destination.java:16]: Added private to the destination and shipping cost class variables for best practices

File: src/main/java/com/shippingcostestimator/enterprise/dto/ShippingCost.java [hunk 2]
```diff
@@ -14,8 +14,8 @@
...
 public @Data
 class ShippingCost {
-    int totalCostId;
-    int serviceLevel;
-    int rates;
-    int estArrival;
+    private int totalCostId;
+    private int serviceLevel;
+    private int rates;
+    private int estArrival;
```
Comment (ATMueller) [inline src/main/java/com/shippingcostestimator/enterprise/dto/ShippingCost.java:20]: Added private to the destination and shipping cost class variables for best practices

File: src/main/resources/main.css [hunk 3]
```diff
@@ -97,7 +97,7 @@
 }
 /*#endregion*/
 
-#btnCalculate{
+#btnCalculate {
```
Comment (ATMueller) [inline src/main/resources/main.css:100]: Formatting changes to keep code consistent

File: src/main/resources/templates/start.html [hunk 4]
```diff
@@ -23,119 +23,123 @@
 </head>
 <body>
 <!--#region Package information -->
-<h4 id="packageInfoHeader">Package Information</h4>
-<!--Carrier select dropdown-->
-<div class="input-group mb-3" id="carrierSelect">
-    <div class="input-group-prepend">
-        <label class="input-group-text" for="carrierSelect">Carrier</label>
+<div id="packageInfoDiv">
```
Comment (ATMueller) [inline src/main/resources/templates/start.html:26]: Added divs to Package Info, Starting Address, and Delivery Address for better organization. (There is too much on the screen at once, you can use these divs to hide what is unnecessary at the time and unhide the divs as they become necessary.)

File: src/main/java/com/shippingcostestimator/enterprise/dto/Destination.java (hunk 1 above)
Comment (discospiff) [reply src/main/java/com/shippingcostestimator/enterprise/dto/Destination.java:16]: Good.

File: src/main/resources/templates/start.html (hunk 4 above)
Comment (discospiff) [reply src/main/resources/templates/start.html:26]: Good way to organize things.

**Comment by discospiff** [comment]
Several good changes here, which reduce technical debt.  I recommend merging this branch.

**Pull request merged**